cmd/vault-plugin-database-elasticsearch/main.go

```diff
@@ -13,7 +13,7 @@ func main() {
 	flags := apiClientMeta.FlagSet()
 	flags.Parse(os.Args[1:])
 
-	if err := elasticsearch.Run(apiClientMeta.GetTLSConfig()); err != nil {
+	if err := elasticsearch.Run(); err != nil {
```

Comment:
Since we're punting on `AutoMTLS` for the time being, this line should be reverted

Reply:
done in 507cc5319bdb4adbd706ad866cd591d087de9dfc